### DISCUSSION TITLE
Hosting Website URL

### DISCUSSION THREAD

**Discussion by Orgbrat** [POPWorldMedia/POPForums#224] [Q&A]
I am using your Open Source PopForums  v17.0.0 Nuget packages hosted within my ASP.NET Core 5.0  web site. So far everything looks and works great but I need to ask a question. Is there anyway to make the Forum title a URL so that my users can return back to the hosting web site home page?

Orgbrat

**Comment by jeffputz** [comment]
I don't understand the question. Can you give an example?

**Reply by Orgbrat** [reply]
Hi Jeff,   How about this I am not ready to go to production yet but browse to [Health Manager Product Site](https://orgbratinfo-testing.azurewebsites.net/) and then click on Forums in top menu.  You will be taken to the Forums area but the user has no easy way to then get back to the product site. If the Title on the Forums home page was a href tag the user could just click on it and be taken back to the product site. Could you set the title value in General settings to a href tag?

**Reply by jeffputz** [reply]
Why not just use the same consistent navigation in the layout page that the forum has to live in?

**Comment by Orgbrat** [comment]
Since I am just using the Nuget packages, do you have an example of  having the forum live in a layout page? Cause today I am just using the website url with area Forums like so 
`<a class="nav-link text-dark" asp-area="Forums" asp-page="Forums">Forums</a>`  
But this would be a good solution.

**Reply by jeffputz** [reply]
The sample project shows that in action.
https://github.com/POPWorldMedia/POPForums.Sample

**Comment by Orgbrat** [comment]
Thanks for all your help
